OneCommunityGlobal/HGNRest
# Ivy-rolePresetsController-create-delete-update-get-test-integration

# Description
Please include the exact bug/functionality description and a summary of the changes/ related issues. Please also include any other relevant motivation and context:
Fixes # (bug list priority high/medium/low x.y.z)
Or Implements # (WBS) 

## Related PRS (if any):
This frontend PR is related to the #XXX backend PR.
To test this backend PR you need to checkout the #XXX frontend PR.
…

## Main changes explained:
Add rolePresetsController.spec.js
Add rolePresetRouter.test.js
Modify rolePresetsController.js
Add createNewPresets.md
Add getPresetByRole.md
Add updatePresetById.md
Add deletePresetsById.md

…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. `npm run test rolePreset`

## Screenshots or videos of changes:
![Screenshot 2024-06-03 at 9 59 30 AM](https://github.com/OneCommunityGlobal/HGNRest/assets/102696148/7e4517ab-5576-48f9-8a56-0041d6d3aa54)


## Note:
Include the information the reviewers need to know.
